### PR TITLE
Update to java 11 (dockerfile and pom.xml)

### PR DESCRIPTION
**Pull request by AlvaroVega**

### CODE REVIEW

**Comment by AlvaroVega** [comment]
`
Exception in thread "main" java.lang.NoClassDefFoundError: javax/xml/bind/JAXBException
	at java.base/java.lang.Class.forName0(Native Method)
	at java.base/java.lang.Class.forName(Class.java:398)
	at org.jboss.logging.Logger$1.run(Logger.java:2252)
	at java.base/java.security.AccessController.doPrivileged(Native Method)
	at org.jboss.logging.Logger.getMessageLogger(Logger.java:2227)
	at org.jboss.logging.Logger.getMessageLogger(Logger.java:2214)
	at org.hibernate.validator.internal.util.logging.LoggerFactory.make(LoggerFactory.java:29)
	at org.hibernate.validator.internal.util.Version.<clinit>(Version.java:27)
	at org.hibernate.validator.internal.engine.ConfigurationImpl.<clinit>(ConfigurationImpl.java:66)
	at org.hibernate.validator.HibernateValidator.createSpecializedConfiguration(HibernateValidator.java:37)
	at org.hibernate.validator.HibernateValidator.createSpecializedConfiguration(HibernateValidator.java:34)
	at javax.validation.Validation$ProviderSpecificBootstrapImpl.configure(Validation.java:220)
	at io.dropwizard.setup.Bootstrap.<init>(Bootstrap.java:66)
	at io.dropwizard.Application.run(Application.java:67)
	at es.tid.fiware.iot.ac.AcService.main(AcService.java:59)
Caused by: java.lang.ClassNotFoundException: javax.xml.bind.JAXBException
	at java.base/jdk.internal.loader.BuiltinClassLoader.loadClass(BuiltinClassLoader.java:581)
	at java.base/jdk.internal.loader.ClassLoaders$AppClassLoader.loadClass(ClassLoaders.java:178)
	at java.base/java.lang.ClassLoader.loadClass(ClassLoader.java:522)
	... 15 more
Exception in thread "main" java.lang.NoClassDefFoundError: javax/xml/bind/JAXBException
	at java.base/java.lang.Class.forName0(Native Method)
	at java.base/java.lang.Class.forName(Class.java:398)
	at org.jboss.logging.Logger$1.run(Logger.java:2252)
	at java.base/java.security.AccessController.doPrivileged(Native Method)
	at org.jboss.logging.Logger.getMessageLogger(Logger.java:2227)
	at org.jboss.logging.Logger.getMessageLogger(Logger.java:2214)
	at org.hibernate.validator.internal.util.logging.LoggerFactory.make(LoggerFactory.java:29)
	at org.hibernate.validator.internal.util.Version.<clinit>(Version.java:27)
	at org.hibernate.validator.internal.engine.ConfigurationImpl.<clinit>(ConfigurationImpl.java:66)
	at org.hibernate.validator.HibernateValidator.createSpecializedConfiguration(HibernateValidator.java:37)
	at org.hibernate.validator.HibernateValidator.createSpecializedConfiguration(HibernateValidator.java:34)
	at javax.validation.Validation$ProviderSpecificBootstrapImpl.configure(Validation.java:220)
	at io.dropwizard.setup.Bootstrap.<init>(Bootstrap.java:66)
	at io.dropwizard.Application.run(Application.java:67)
	at es.tid.fiware.iot.ac.AcService.main(AcService.java:59)
Caused by: java.lang.ClassNotFoundException: javax.xml.bind.JAXBException
	at java.base/jdk.internal.loader.BuiltinClassLoader.loadClass(BuiltinClassLoader.java:581)
	at java.base/jdk.internal.loader.ClassLoaders$AppClassLoader.loadClass(ClassLoaders.java:178)
	at java.base/java.lang.ClassLoader.loadClass(ClassLoader.java:522)
	... 15 more
INFO: keypass entrypoint start
INFO: LOG LEVEL <INFO>
INFO: MySQL endpoint <iot-mysql>
INFO: KEYPASS_DB_HOST_NAME <iot-mysql>
INFO: KEYPASS_DB_HOST_PORT <3306>
INFO: KEYPASS_DB_USER <keypass>
INFO: KEYPASS_DB_PASSWORD <keypass>
iot-mysql [172.17.0.6] 3306 (?) open
`

File: ChangeLog [hunk 1]
```diff
@@ -1,3 +1,4 @@
+UPGRADE: Java from 8 to 11
```
Comment (fgalan) [inline ChangeLog:1]:
```suggestion
UPGRADE: Java from 8 to 11 in Dockerfile
```

The other UPGRADE below refer to pom.xml and maybe the difference could be important

**Comment by AlvaroVega** [comment]
This PR is ready to be merged

File: Dockerfile [hunk 2]
```diff
@@ -17,11 +17,11 @@ WORKDIR /opt/keypass
 
 RUN \
     # openjdk-8-jdk is not included in Debian 11 repositories, so we need to add jessie-backports for it
```
Comment (fgalan) [inline Dockerfile:19]:
```suggestion
```

File: Dockerfile [hunk 3]
```diff
@@ -17,11 +17,11 @@ WORKDIR /opt/keypass
...
-    echo 'deb http://httpredir.debian.org/debian-security stretch/updates main' > /etc/apt/sources.list.d/jessie-backports.list && \
+    #echo 'deb http://httpredir.debian.org/debian-security stretch/updates main' > /etc/apt/sources.list.d/jessie-backports.list && \
```
Comment (fgalan) [inline Dockerfile:20]:
```suggestion
```

**Review by fgalan — APPROVED** [review]
LGTM

**Pull request merged**